lib/authz/permissions.go

```diff
@@ -1043,6 +1043,23 @@ func definitionForBuiltinRole(clusterName string, recConfig types.SessionRecordi
 						types.NewRule(types.KindClusterAuthPreference, services.RO()),
 						types.NewRule(types.KindRole, services.RO()),
 						types.NewRule(types.KindLock, services.RW()),
+						// Okta can manage access lists and roles it creates.
+						{
+							Resources: []string{types.KindRole},
+							Verbs:     services.RW(),
+							Where: builder.Equals(
+								builder.Identifier(`resource.metadata.labels["`+types.OriginLabel+`"]`),
+								builder.String(types.OriginOkta),
+							).String(),
+						},
+						{
+							Resources: []string{types.KindAccessList},
```

Comment:
Could you write a UT for `types.KindAccessList` and  [(s *Service) ListAccessLists(](https://github.com/gravitational/teleport.e/blob/d6c4847b42bcbfe91e0f0b9bba52e4f7fab63750/lib/accesslist/service.go#L219) flow.